src/transform/index.ts

```diff
@@ -7,11 +7,14 @@ import { transformResponsesObj, transformRequestBodies } from "./responses";
 import { transformSchemaObjMap } from "./schema";
 
 interface TransformOptions {
+  immutableTypes: boolean;
   rawSchema?: boolean;
   version: number;
 }
 
-export function transformAll(schema: any, { version, rawSchema }: TransformOptions): string {
+export function transformAll(schema: any, { immutableTypes, rawSchema, version }: TransformOptions): string {
+  const readonly = immutableTypes ? "readonly " : "";
```

Comment:
Not a blocker for this PR, but I’d suggest moving this into `src/utils.ts` as `tsReadonly()` util. Similar to the existing `tsUnionOf()` util and others.

It’s a minor thing, but it makes modifications quicker, and helps prevent typos.

Reply:
no problem, I can do that quick as well. I had it set up originally as a function, but changed it back.